### ISSUE TITLE
Application logger doesn't send any emails

### ISSUE THREAD

**Issue by alexbaat** [pimcore/pimcore#8150]
# Bug Report

### Expected behavior
In the System Setting you can activate 'Send log summary via email' for the Application Logger. The sending of the emails seems to be handled bij the maintenance tasks logmaintenance and checkerrorlogsdb.
When these tasks are run, an email should be sent to the set emailadresses for the logs of a certain log level/priorty.

### Actual behavior
No emails are sent.

### Steps to reproduce
Activate  'Send log summary via email' for the Application Logger and fill in an emailadress. Set a certain priority.
Now make sure some logs are created and then run the maintenance tasks.

### Code example
In the lib/Maintenance/Tasks/LogMailMaintenanceTask.php I see this piece of code
```php
            $logLevel = (int) ($this->config['applicationlog']['mail_notification']['filter_priority'] ?? null);

            $query = 'SELECT * FROM '.ApplicationLoggerDb::TABLE_NAME." WHERE maintenanceChecked IS NULL AND priority <= $logLevel order by id desc";
```
As you can see it is checking the priority as an integer.However the 'priority' field in the database is an enum field with the values 'emergency','alert','critical','error','warning','notice','info','debug'. Now in my test I set the log level to ERROR in the system settings and this uses priorty 3. So the current query is
```
SELECT * FROM application_logs WHERE maintenanceChecked IS NULL AND priority <= 3 order by id desc
```
but this will not return any records because the value in the database is 'error' and not 3.

Same problem seems to be in the file lib/Maintenance/Tasks/CheckErrorLogsDbTask.php. However I don't know why this tasks does actually the same. It seems duplicate. 

# Feature Request

**Comment by danielschmieder** [comment]
Is there a plan to fix this issue? Would be great to have this feature again. :)

**Comment by brusch** [comment]
@danielschmieder @alexbaat can anyone provide a PR that fixes the issue? That would be great and speed things up - thanks in advance!

**Comment by danielschmieder** [comment]
@brusch I could provide my piece of code helping me temporarily fixing it (for me). The easiest way could be just adjusting the execute-function in LogMailMaintenanceTask. Maybe someone can help out providing a PR?

```
# lib/Maintenance/Tasks/LogMailMaintenanceTask.php

$logLevel = /*(int)*/ ($this->config['applicationlog']['mail_notification']['filter_priority'] ?? null);

$logLevels = ['debug', 'info', 'notice', 'warning', 'error', 'critical', 'alert', 'emergency'];
$firstLogLevel = array_search($logLevel, $logLevels);
$relevantLogLevels = array_slice($logLevels, $firstLogLevel);
$priorityList = implode('\',\'', $relevantLogLevels);

$query = 'SELECT * FROM '.ApplicationLoggerDb::TABLE_NAME." WHERE maintenanceChecked IS NULL AND priority IN ('$priorityList') order by id desc";
```

and because arrays are not allowed any more:

`$mail->addTo(implode(',', $receivers));`

Hope it helps a bit.

**Comment by mattamon** [comment]
Fixed by #17596
